### PR TITLE
Update capacity-planning.md

### LINKED ISSUE #2337: Message "unsupported file system" in the log when using supported XFS file system
### Describe the bug

When starting QuestDB with the data directory on my XFS file system, the log shows "UNSUPPORTED FILE SYSTEM" and "EXPERIMENTAL", despite the docs saying xfs is supported (see https://questdb.io/docs/operations/capacity-planning/#storage-and-filesystem).

```
2022-07-20T08:53:44.789850Z A server-main QuestDB server 6.4.3. Copyright (C) 2014-2022, all rights reserved.^M
2022-07-20T08:53:44.889796Z I server-main web console is up to date^M
2022-07-20T08:53:44.890318Z A server-main Server config: /home/stefan/.questdb/conf/server.conf^M
2022-07-20T08:53:44.919858Z I i.q.c.t.t.InputFormatConfiguration loading input format config [resource=/text_loader.json]^M
2022-07-20T08:53:44.931846Z A server-main Config changes applied:^M
2022-07-20T08:53:44.931873Z A server-main   http.enabled : true^M
2022-07-20T08:53:44.931894Z A server-main   tcp.enabled  : true^M
2022-07-20T08:53:44.931912Z A server-main   pg.enabled   : true^M
2022-07-20T08:53:44.931925Z A server-main open database [id=1435148172356326765.4686310487359770726]^M
2022-07-20T08:53:44.931961Z A server-main platform [bit=64]^M
2022-07-20T08:53:44.931981Z A server-main OS/Arch: linux/amd64 [AVX2,8]^M
2022-07-20T08:53:44.933800Z A server-main available CPUs: 16^M
2022-07-20T08:53:44.933826Z A server-main db root: /home/stefan/.questdb/db^M
2022-07-20T08:53:44.933847Z A server-main backup root: null^M
2022-07-20T08:53:44.933917Z A server-main db file system magic: 0x58465342 [XFS] EXPERIMENTAL^M
2022-07-20T08:53:44.933965Z A server-main


                        *** SYSTEM IS USING UNSUPPORTED FILE SYSTEM AND COULD BE UNSTABLE ***

^M
2022-07-20T08:53:44.934558Z A server-main SQL JIT compiler mode: on^M
[...]
```

### To reproduce

1. `./questdb.sh start`, with the data directory residing on an XFS file system
2. Inspect beginning of log file

### Expected Behavior

No message about unsupported file system in the log.
Alternatively, message about XFS being a supported file system and "all good".

### Environment

```markdown
- **QuestDB version**: 6.4.3
- **OS**: Linux desktop 5.15.0-41-generic #44-Ubuntu SMP Wed Jun 22 14:20:53 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Rajakavitha1**
- Fixes https://github.com/questdb/questdb/issues/2337

### CODE REVIEW

File: docs/deployment/capacity-planning.md [hunk 1]
```diff
@@ -34,15 +32,13 @@ QuestDB officially supports the following filesystems:
 - EXT4
 - NTFS
 - OVERLAYFS (used by Docker)
```
Comment (goodroot) [inline docs/deployment/capacity-planning.md:34]:
```suggestion
- NTFS
- OVERLAYFS (used by Docker)
- XFS (`ftype=1` only)
```

Comment (goodroot) [reply docs/deployment/capacity-planning.md:34]: XFS will work if this is set!

**Review by goodroot — COMMENTED** [review]
Thanks @Rajakavitha1 ! 

Minor clarifications.

**Comment by Rajakavitha1** [comment]
> Thanks @Rajakavitha1 !
> 
> Minor clarifications.

Done!!

**Comment by javier** [comment]
thanks!

**Pull request merged**